From 6b13301897bb675f4cd15268614977c3235cd280 Mon Sep 17 00:00:00 2001
From: Rahul Anand <ranand@codeaurora.org>
Date: Thu, 27 Feb 2014 21:37:47 +0000
Subject: [PATCH] Assignment of O2 changed to '=' to remove warnings The
 checked out Makefile.am was using := causing automake to generate warnings,
 which were being treated as errors by autoreconf

Change-Id: Ib6974d84cc8dffe3947c371c3bb98e4a9e780e86
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 511c4b7..661afe8 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,7 +1,7 @@
 ACLOCAL_AMFLAGS = -I m4
 
 bin_PROGRAMS = mpdecision
-mpdecision_CFLAGS := -O2
+mpdecision_CFLAGS = -O2
 mpdecision_CFLAGS += -g
 mpdecision_CFLAGS += -Wall
 mpdecision_CFLAGS += -Werror
-- 
1.8.1.2

